edfio/edf.py

```diff
@@ -996,6 +998,130 @@ def _set_startdate_with_recording(self, recording: Recording) -> None:
         except AnonymizedDateError:
             self._startdate = datetime.date(1985, 1, 1)
 
+    @property
+    def data_record_duration(self) -> float:
+        """Duration of each data record in seconds."""
+        return self._data_record_duration
+
+    def update_data_record_duration(
+        self,
+        data_record_duration: float,
+        method: Literal["strict", "pad", "truncate"] = "strict",
+    ) -> None:
+        """
+        Update the data record duration.
+
+        This operation will fail if the new duration is incompatible with the current
+        sampling frequencies.
+
+        Parameters
+        ----------
+        data_record_duration : float
+            The new data record duration in seconds.
+        method : `{"strict", "pad", "truncate"}`, default: `"strict"`
+            How to handle the case where the new duration does not divide the Edf
+            duration evenly
+
+            - "strict": Raise a ValueError
+            - "pad": Pad the data with zeros to the next compatible duration. If zero
+               is outside the physical range, data is padded with the physical minimum.
```

Comment:
This fixes doc formatting:

```suggestion
              is outside the physical range, data is padded with the physical minimum.
```